src/main/scala/me/rexim/morganey/ReplAutocompletion.scala

```diff
@@ -127,18 +130,34 @@ object ReplAutocompletion {
     }
   }
 
-  private object IsCommand {
+  private object SimpleCommand {
 
     def unapply(line: String): Option[List[String]] = {
       val potentialCommand = parseCommand(line).map(_._1)
       val matchingCommands = potentialCommand map { p =>
-        commands.keySet filter (_ startsWith p)
+        commands.values collect {
+          case c if c.name startsWith p =>
+            if (c.hasTermArgument) c.name + " " else c.name
```

Comment:
I don't understand this code. Could you please elaborate on what's going on here? Why do we care about term arguments whe we parse a simple command?